plugin/evm/vm.go

```diff
@@ -706,6 +704,7 @@ func (vm *VM) initializeChain(lastAcceptedHash common.Hash) error {
 		node,
 		&vm.ethConfig,
 		vm.createConsensusCallbacks(),
+		&EthPushGossiper{vm: vm},
```

Comment:
It's a bit dubious that `vm.ethTxPushGossiper` isn't initialized yet

Reply:
We now drop txs if not initialized.

Reply:
I think that if check would be racy... but like... 🤷. I feel like we could just document the assumption that the API can't be hit before bootstrapping has finished.